desktop/components/credit_card/test/error_handling_form.coffee

```diff
@@ -66,3 +66,13 @@ describe 'FavoritesStatusModalView', ->
     it 'adds stripe errors', ->
       @errorHandlingForm.showError 'description', { status: 400, error: { additional: 'additional info'} }
       $('.error').text().should.equal 'Your card appears to be missing or malformed. Please try another card or contact support. additional info'
+
+    it 'handles @errors properties that are a function (that return an errors object)', ->
```

Comment:
this is to get the proper `this` when we are passing a function rather than a string as an `errors` property.